comit/src/ledger.rs

```diff
@@ -1,5 +1,18 @@
 use crate::ethereum::ChainId;
+use fmt::Display;
 use serde::{de::Error as _, Deserialize, Deserializer, Serialize, Serializer};
+use std::fmt;
+
+/// The ledger network kind. We define this as a cross blockchain domain term.
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum Kind {
```

Comment:
I like this!

Maybe we can even go further in the future and instead of just classifying the networks of other ledgers, actually establish a direct mapping of networks on COMIT to networks of other ledgers.

Reply:
I've wanted to introduce this for pretty much as long as I can remember while working at CoBloX :)

Reply:
I reckon we should have done that way earlier but it always felt a bit weird without lifting the abstraction level of COMIT on layer up (from swaps to trading).